.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+
+language: cpp
+
+services:
+- docker
+
+install: skip
+
+script:
+- echo "skipping tests"
+- docker login --username ${DOCKER_USERNAME} --password ${DOCKER_PASSWORD}
+- docker pull ${DOCKER_USERNAME}/cjio:latest || true
+- docker build . --cache-from ${DOCKER_USERNAME}/cjio:latest -t ${DOCKER_USERNAME}/cjio:latest
+
+
+deploy:
+- provider: script
+  script:
+    - docker push ${DOCKER_USERNAME}/cjio:latest
```

Comment:
The 'script' provider requires a string, not an array, so this will fail.
The solution is:
```
script: docker push ...
```